app/templates/components/gh-theme-error-li.hbs

```diff
@@ -0,0 +1,13 @@
+<li>
```

Comment:
True but for purely presentational components like this I prefer to keep all of the markup inside the template, that way there's no need to look in both .js and .hbs files if looking to change things